.github/workflows/sha2.yml

```diff
@@ -21,7 +21,7 @@ jobs:
   set-msrv:
     uses: RustCrypto/actions/.github/workflows/set-msrv.yml@master
     with:
-        msrv: 1.72.0
+      msrv: 1.79.0
```

Comment:
This seems like an aggressive bump for now but I imagine it will be awhile before we actually get a final version of this crate out